WordPress/Sniffs/WP/UseCapabilitiesNotRolesSniff.php

```diff
@@ -0,0 +1,234 @@
+<?php
+/**
+ * WordPress Coding Standard.
+ *
+ * @package WPCS\WordPressCodingStandards
+ * @link    https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards
+ * @license https://opensource.org/licenses/MIT MIT
+ */
+
+namespace WordPress\Sniffs\WP;
+
+use WordPress\AbstractFunctionParameterSniff;
+
+/**
+ * User capabilities should be used not roles.
+ *
+ * @link    https://make.wordpress.org/themes/handbook/review/required/#options-and-settings
+ *
+ * @package WPCS\WordPressCodingStandards
+ *
+ * @since   1.0.0
+ */
+class UseCapabilitiesNotRolesSniff extends AbstractFunctionParameterSniff {
+
+	/**
+	 * The group name for this group of functions.
+	 *
+	 * @since 1.0.0
+	 *
+	 * @var string
+	 */
+	protected $group_name = 'caps_not_roles';
+
+	/**
+	 * List of functions that accept roles and capabilities as an agrument.
+	 *
+	 * The number represents the position in the function call
+	 * passed variables, here the capability is to be listed.
```

Comment:
`The number represents the position in the function call passed variables, here the capability is to be listed.`

:point_up:  This sentence seems a bit wonky.

Did you mean: `The number represents the parameter position in the function call, where the capability is supposed to be listed.` ?